mava/components/jax/training/trainer.py

```diff
@@ -110,6 +128,7 @@ def on_building_init_end(self, builder: SystemBuilder) -> None:
                     matches = most_matches
                     builder.store.table_network_config[trainer_key] = sample
 
+        # TODO (Matthew): networks need to be created on the nodes instead?
```

Comment:
Were you able to reach a conclusion on this?

Reply:
Afraid not, I think this ties into the Jax network init issue though. Do we currently have an issue out for this?
I don't want to remove the comment if we do not.

Reply:
I am not sure whether is an issue out yet. Please keep in that case.